### PR TITLE
Add news section

### PR DESCRIPTION
**Pull request by 76creates**
Goal: add news page
Stuff done:
- added news page, and article page
- converted urls paths to absolute due to multi page 
- added "Running in local" section to README.md
- tidying up: removed jekyll-cache, and kunsteriks dirs, added them to gitignore as well as _site dir
- minor updates

Left placeholder article as a "guide", when the first article is written it should be removed

### CODE REVIEW

**Review by mariannovotny — APPROVED** [review]

**Comment by mariannovotny** [comment]
@76creates  Looks good to me, can you please resolve merge conflict and merge it?
Thanks

File: docs/README.md [hunk 1]
```diff
@@ -1 +1,22 @@
...
+~~~
+---
+layout: article
+title: title here
+categories: news
+excerpt_separator: <!--more-->
+image: image filename here
+---
+~~~
+- Title image should be added to `/media/news/title` directory.
```
Comment (ljubon) [inline docs/README.md:21]: I hate myself while writing this, but `tittle` with 2x `t` (name of direct and description)
I know you just used the name of the existing directory but it's no big thing to change 🙏

**Comment by ljubon** [comment]
hey @76creates really clean PR, could you just please squash these commits? 
imho we should keep clean git history and this PR can be threated as one change 🙏 
other than that, no other comments 🚀

**Comment by 76creates** [comment]
> @76creates Looks good to me, can you please resolve merge conflict and merge it?
> Thanks

resolved, its ready for merging

**Comment by 76creates** [comment]
> hey @76creates really clean PR, could you just please squash these commits once conflict is resolved?
> imho we should keep clean git history and this PR can be threated as one change 🙏
> other than that, no other comments 🚀

folk that merges should chose "squash and merge" if its repo policy, I agree with you and I did merge initial commit after testing tho ⚡

**Pull request merged**